packages/rule-components/src/ReportDetails/RiskDescription.js

```diff
@@ -8,10 +8,13 @@ const RiskDescription = ({ riskValue, riskMeta }) => {
 
     return (
         <div>
-            <div>
+            <div className="battery">
```

Comment:
Where is this class comming from?

Reply:
hm, it's from uhc-portal, I will create similar one in frontend components

Reply:
fixed